docs/docs/benchmark.md

```diff
@@ -94,3 +124,43 @@ This is done by:
 <li><a href="new/study">Creating a new study</a> with the name of the benchmark as the alias.</li>
 <li>Adding new tasks to the study by <i>tagging</i> them with the name of the benchmark.</li>
 </ul>
+
+Detailed steps:
+- Create a new study with name and description
+- Set flag to "private" - we dont want people adding tasks here
+- Set "suite" flag
+- Find the study tag study_X
+- Tag all tasks with study_X
+- You can add other tags as well. For each you can set:
+    - User IDs (who can extend study with that tag)
+    - Beginning/end dates (only stuff tagged within those dates are part of the study)
+- You do not have to tag any runs with study_X, these will be linked automatically
+  
+### Changing a benchmark suite
+- If the study is private, only the owner can do that
+- You can create a new suite with a similar set of tasks if you want
+
+
+## Existing suites
+
+### OpenML-CC18
+https://www.openml.org/s/99
+
+### OpenML100  
+https://www.openml.org/s/14
+
+The OpenML100 was a predecessor of the OpenML-CC18, consisting of <a href="https://www.openml.org/search?q=tags.tag%3AOpenML100&type=data&table=1&size=100" target="_blank">100 classification datasets</a>. We recommend that you use the OpenML-CC18 instead, because the OpenML100 suffers from some teething issues in the design of benchmark suites. For instance, it contains several datasets that are too easy to model with today's machine learning algorithms, as well as datasets that represent time series analysis problems. These do not invalidate benchmarks run on the OpenML100, but may obfuscate the interpretation of results. The 'OpenML-CC18' handle is also more descriptive and allows easier versioning.
+
+For reference, the OpenML100 included datasets satisfying the following requirements:
+<ul><li>the number of observations are between 500 and 100000 to focus on medium-sized datasets, that are not too small for proper training and not too big for practical experimentation,
```

Comment:
minor, but for clarity i would indent this as following: 

<ul>
  <li> ... </li>
  etc ... 
</ul>